package.json

```diff
@@ -58,18 +62,15 @@
     "jest": "^21.1.0",
     "nodemon": "^1.12.1",
     "npm": "^5.4.2",
+    "react": "^15.6.1",
     "react-dev-utils": "^4.0.1",
+    "react-dom": "^15.6.1",
     "react-test-renderer": "^15.6.1",
     "webpack": "^3.6.0",
     "webpack-bundle-analyzer": "^2.9.0",
     "webpack-dashboard": "^1.0.0-6",
     "webpack-dev-server": "^2.8.2",
-    "webpack-merge": "^4.1.0",
-    "@storybook/react": "^3.2.10",
-    "@storybook/addon-actions": "^3.2.10",
-    "@storybook/addon-links": "^3.2.10",
-    "react": "^15.6.1",
-    "react-dom": "^15.6.1"
+    "webpack-merge": "^4.1.0"
   },
   "peerDependencies": {
     "react": "^15.6.1",
```

Comment:
mentioned it in a previous PR but you should relax the `peerDependencies` to `>= 15` IMO. That way it works with all v15 and not just the latest version of 15. For instance we wouldn't be able to use this in EDS because we're on 15.4.1

Reply:
Thanks! Makes sense, changing it!